.github/labeler.yml

```diff
@@ -1,8 +1,27 @@
-version: 1
+version: 2
 labels:
+# type 1: package-action labels
 - label: "Update Package"
   title: '^upd\(.*'
+- label: "Update Pacscript"
+  title: '^fix\(.*'
+- label: "bug"
+  title: '^bug\(.*'
+- label: "enhancement"
+  title: '^enh\(.*'
+- label: "revert"
+  title: '^rev\(.*'
+
+# type 2: repo-action labels
 - label: "Package Add"
   title: "^add:.*"
 - label: "remove package"
   title: "^rm:.*"
+- label: "chore"
+  title: "^chore:.*"
+- label: "CI"
+  title: "^ci:.*"
+- label: "meta"
+  title: "^meta:.*"
```

Comment:
```suggestion
```
What would a meta PR be for?

Reply:
an overall change to the pacstall-programs repo or to how pacstall functions

Reply:
Then wouldn't that be a feat?

Reply:
Then I'll make one.

Reply:
I can swap this to `feat` when you do